src/components/grammar-courses/basic-pronouns-articles/LessonComponent.jsx

```
@@ -0,0 +1,334 @@
+
+    setScore(currentScore);
+    setShowResults(true);
+  };
+
+  const resetExercise = () => {
+    setSelectedAnswers({});
+    setShowResults(false);
+    setScore(0);
+  };
+
+  const isAnswerCorrect = (exerciseIndex, questionIndex) => {
```

ESLint

[error] 'isAnswerCorrect' is assigned a value but never used.

(no-unused-vars)